### ISSUE TITLE
Intervalometer mode for DSLR for time lapses

### ISSUE THREAD

**Issue by flyingbarron** [artyom-beilis/OpenLiveStacker#64]
For DSLRs, instead of streaming the images, allow the creation of a schedule of images to be taken

e.g. 
take 100 images of 10 second exposure with a 5 second delay between images.

This will enable the creation of time lapse images. 

An optional, more advanced, feature would be a sequence of images with control over exposure, so that the image sequence can be done for optimal foreground and background in a single sequence.

**Comment by artyom-beilis** [comment]
I'm not even sure there is an API for this in gphoto2.

Still most of the stuff should be configured via camera itelf.

**Comment by flyingbarron** [comment]
No, the intervalometer/timer/scheduler would be implemented in OLS itself.
It would just trigger the camera/gphoto2 on a schedule instead of constantly.

**Comment by artyom-beilis** [comment]
But why would you skip frames? Don't you want as much data as possible?

**Comment by artyom-beilis** [comment]
> If it's a time-lapse, you don't necessarily need all the frames -> this is not for real time stacking, it's to create a time lapse video.

So basically you don't do any processing? Just recording?

**Comment by flyingbarron** [comment]
Correct.
You could also create the timelapse in OLS, but that would be another issue/enhancement request.
